src/ui/layout/qgssvgexportoptions.ui

```diff
@@ -36,7 +36,7 @@
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="mIncludeMetadataCheckbox">
         <property name="text">
-         <string>Export RDF metadata</string>
+         <string>Export RDF metadata (title, author, etc.)</string>
```

Comment:
How about as a tooltip (allowing to be more verbose)? Keeping the label short as is would avoid us updating screenshots in docs. Also these information could be backported without affecting the UI.

Reply:
@DelazJ , I recently faced a situation where someone's safety was put at risk because he or she did not realize he/she was exporting author metadata. A tooltip is not visible enough.

Plus, it really only adds three word to a label, this isn't abusive by any definition of over-verbose :wink:

Reply:
Most of the options in this dialog are not self-explanatory and are showing long tooltip (btw, they could be worth wrapping on multiple lines). And if I were about to use an option I'm not sure of, I think checking whether there's a minimal explanation in tooltip should be the minimal step. No?
> Plus, it really only adds three word to a label, this isn't abusive by any definition of over-verbose 

Sure. By verbose, I meant you could add more information on which metadata is exported (or not) in the tooltip. And deprecates the screenshot.

Reply:
The screenshot here is already deprecated due to the additional checkbox added to control georeferencing.

I like your idea of adding the actual metadata strings into the tooltip, might do that later. That said, this recent security incident convinced the label needed a bit more context outright.

Reply:
> The screenshot here is already deprecated due to the additional checkbox added to control georeferencing.

True, for PDF (!). But I agree the option should show the same label in both dialogs.
My actual concern is that we are "understaffed" for [screenshot updates](https://github.com/qgis/QGIS-Documentation/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3AScreenshots+) in docs (in docs, _tout court_, but another story...) so generally speaking, if we can avoid GUI changes when it's not necessary...
My 2cts.

Reply:
@DelazJ , agreed on avoiding unnecessary burden. In this case, it's both doc and translation teams impacted. But in this case, the increased security through in-your-face clarification warrants this extra burden I'm afraid.

Valuable 2cts, keep those discussions alive :)